.github/workflows/release.yaml

```
@@ -18,14 +20,20 @@ jobs:
         uses: kubernetes-sigs/release-actions/setup-tejolote@dd08496c83441d6477114cc0555b96d404dacff7 # v0.1.2
 
       - name: Run tejolote 
-        env:
-          TAG: ${{ github.ref_name }}
         run: |
-          tejolote attest --artifacts github://kubernetes-sigs/karpenter/"${TAG}" github://kubernetes-sigs/karpenter/"${{ github.run_id }}" --output karpenter.intoto.json --sign
+          tejolote attest --artifacts github://kubernetes-sigs/karpenter/${{ env.TAG }} github://kubernetes-sigs/karpenter/"${{ github.run_id }}" --output karpenter.intoto.json --sign
```

```suggestion
          tejolote attest --artifacts "github://kubernetes-sigs/karpenter/$TAG" "github://kubernetes-sigs/karpenter/${{ github.run_id }}" --output karpenter.intoto.json --sign
```

This syntax avoids the script injection possibility

---

Since this has been changed in https://github.com/kubernetes-sigs/karpenter/commit/d4ff37c70609c2da3afa85924707ecaef1e71a22, I've rebased the PR. Now it only has the changes related to openvex.